toplevel/coqtop.ml

```diff
@@ -456,6 +456,9 @@ let init_toplevel custom_init arglist =
       Flags.if_verbose print_header ();
       Mltop.init_known_plugins ();
       Global.set_engagement opts.impredicative_set;
+      Global.set_uip opts.uip;
+      (Global.set_allow_sprop opts.allow_sprop);
+      (if opts.cumulative_sprop then Global.make_sprop_cumulative ());
```

Comment:
It would be cool if we could aggregate all the consistency-related options into a single record, right?